client/reader/stream/index.jsx

```diff
@@ -164,7 +164,7 @@ export default class ReaderStream extends React.Component {
 			recs,
 			updateCount: store.getUpdateCount(),
 			selectedIndex: store.getSelectedIndex(),
-			isFetchingNextPage: store.isFetchingNextPage(),
+			isFetchingNextPage: store.isFetchingNextPage && store.isFetchingNextPage(),
```

Comment:
hmmm... so ... we have a valid store but it doesn't have a `isFetchingNextPage` method? That seems odd.